### PR TITLE
Refactor automated build

### PR DESCRIPTION
**Pull request by Ksisu**
#### Short description 
I rewrote the whole automated docker build to use GitHub Actions workflow.

#### Why?
Because it's much faster and nicer :)

#### What else has changed?

- The builds are running in parallel.
- SBT version is getting dynamically from bintray sbt repo (always the latest).
- Runs periodically every day at 00:00 UTC. (So if there a new SBT then next day we have an image with that.)
- Runs on pull requests too but then just build and not push to DockerHub.

#### What has to be done before merge?

- Disable automated build on DockerHub.
- Create a new access-token on DockerHub (https://hub.docker.com/settings/security) and add it to GitHub as secret (https://github.com/hseeberger/scala-sbt/settings/secrets). Secret name: `DOCKER_PASSWORD`

### CODE REVIEW

**Comment by francisdb** [comment]
Cool, that way we'll at least get to see what fails during or builds.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            dockerContext: 'graalvm-ce'
+            baseImageTag: '19.3.0-java11'
+          - javaTag: 'graalvm-ce-19.3.0-java8'
+            dockerContext: 'graalvm-ce'
+            baseImageTag: '19.3.0-java8'
+    steps:
+    - uses: actions/checkout@v1
+    - name: Get latest SBT version
+      id: get_sbt_version
+      run: |
```
Comment (francisdb) [inline .github/workflows/build.yml]: What would the process be when a new sbt version is released? I guess we could as well put it as a variable?

Comment (Ksisu) [reply .github/workflows/build.yml]: The process kind of looks like this:
1. released a new sbt version eg. `1.3.6`
2. the sbt team put it into a linux package repository https://dl.bintray.com/sbt/
3. in the `build.yml:L11` (`- cron:  '0 0 * * *'`) says: "run this workflow every day at midnight"
4. in the workflow the `get_sbt_version` step find out which is the latest sbt version (get all sbt debian packages metadata from here: https://dl.bintray.com/sbt/debian/Packages and filter out the latest version)

So when a new sbt version released we just have to wait one day and automatically build the new image.
But ofc we could use a "static" variable, but then we should update it after every sbt release.

Comment (francisdb) [reply .github/workflows/build.yml]: makes sense, missed that part

**Comment by hseeberger** [comment]
Very cool stuff! Thanks a lot. Will take some time to chew on it but eventually merge it.

**Comment by hseeberger** [comment]
@Ksisu could you please resolve the conflicts which arose after merging other PRs.

**Comment by Ksisu** [comment]
@hseeberger Done :)

**Comment by hseeberger** [comment]
Giving it a try ;-)

**Comment by Ksisu** [comment]
@hseeberger I think the `DOCKER_PASSWORD` secret is not configured (see my first comment).
After then you can click the `Re-run checks` button at https://github.com/hseeberger/scala-sbt/runs/364915954 ;)

**Comment by hseeberger** [comment]
@Ksisu The secret was configured, but too late ;-) I reran and now all builds are green. Really fast! Great work! Thanks a lot!

**Pull request merged**